apps/theming-designer/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html dir="ltr">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>Theming Designer App</title>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/react/16.3.2/umd/react.production.min.js"></script>
+    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/react-dom/16.3.2/umd/react-dom.production.min.js"></script>
```

Comment:
Would be good to target react 16.8.something.

Reply:
If you want to add 16.8 (and matching types) as a dep for just this project, you'll need to add it to common-versions.json allowedAlternateVersions.